docs/sources/writing-guide/front-matter/index.md

```diff
@@ -70,6 +73,44 @@ They do not appear in the resulting HTML source for the page and do not affect S
 
 Ideally, use single terms as opposed to phrases.
 
+### labels
+
+Used to add labels that appear before the topic title.
+Only certain labels are supported.
+
+For `labels.products`, the supported values and the resulting website labels are:
+
+- `cloud`: "Grafana Cloud"
+- `enterprise`: "Enterprise"
+- `oss`: "Open source"
+
+Labels can be inherited through cascading front matter.
+Each project has a set of default labels defined in the root `_index.md` file for the project.
+
+For versioned projects, the `_index.md` file resides in the website repository.
+For unversioned projects, the `_index.md` file resides in the project repository.
+
+If the default labels are incorrect for a page, or directory of pages, you should update the labels.
+A page should include a label in the `labels.products` sequence for every product that the page is relevant to.
+
+For a single page, if the page describes a feature available in Grafana Cloud and Grafana Enterprise, the source file front matter should include the following:
```

Comment:
```suggestion
For a single page, if the page describes a feature available in Grafana Cloud and Grafana Enterprise, the source file front matter looks as follows:
```

Reply:
My concern with the suggested wording is that the following snippet is only part of the front matter. I would expect all of the other usual front matter mappings to exist in addition to the snippet example.